test/Electrolyzer/Period_map.csv

```diff
@@ -0,0 +1,366 @@
+Period_Index,Rep_Period,Rep_Period_Index
+1,72,2
+2,72,2
+3,72,2
+4,72,2
+5,72,2
+6,72,2
+7,72,2
+8,72,2
+9,72,2
+10,72,2
+11,72,2
+12,72,2
+13,72,2
+14,72,2
+15,72,2
+16,72,2
+17,72,2
+18,72,2
+19,72,2
+20,72,2
+21,72,2
+22,72,2
+23,72,2
+24,72,2
+25,72,2
+26,72,2
+27,27,1
+28,72,2
+29,72,2
+30,72,2
+31,72,2
+32,72,2
```

Comment:
This Period_map file seems odd. Is this 365 days being represented by 5 days?

Reply:
Yes.